apps/android_rpc/app/build.gradle

```diff
@@ -59,14 +59,14 @@ tasks.withType(JavaCompile) {
 // }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
     defaultConfig {
         applicationId "org.apache.tvm.tvmrpc"
         minSdkVersion 24
-        targetSdkVersion 26
+        targetSdkVersion 33
```

Comment:
Could you please check if the OpenCL wrapper will successfully work with this version of targetSDK?
I saw the problem with the target sdk > 30 that in the runtime it was impossible to open libOpenCL.so on the device. Probably it will be better to use target sdk == 30 for now and upgrade it later.

Reply:
Egor and I discussed it, and we decided to consider this question separately, and now to merge the PR.